packages/lab/src/list-next/ListItemNext.tsx

```diff
@@ -0,0 +1,91 @@
+import { forwardRef, HTMLAttributes, MouseEvent } from "react";
+import { clsx } from "clsx";
+import { makePrefixer, useId } from "@salt-ds/core";
+
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import listItemNextCss from "./ListItemNext.css";
+import { useListItems } from "./ListNextContext";
+
+const withBaseName = makePrefixer("saltListItemNext");
+
+export interface ListItemNextProps extends HTMLAttributes<HTMLLIElement> {
+  label?: string;
+  /**
+   * If true, the particular list item in list will be disabled.
+   */
+  disabled?: boolean;
+  selected?: boolean;
+  id?: string;
+  value: string;
+}
+
+export const ListItemNext = forwardRef<HTMLLIElement, ListItemNextProps>(
+  function ListItemNext(
+    {
+      children,
+      className,
+      disabled: disabledProp,
+      selected: selectedProp,
+      label,
+      id: idProp,
+      value,
+      onClick,
+      ...props
+    },
+    ref
+  ) {
+    const targetWindow = useWindow();
+    useComponentCssInjection({
+      testId: "salt-list-item-next",
+      css: listItemNextCss,
+      window: targetWindow,
+    });
+    const id = useId(idProp) || value;
+
+    const listContext = useListItems();
```

Comment:
```suggestion
    const listContext = useListItem();
```
I would prefer this was singular since it's in one list item